ansible/roles/artifactory/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Install Python dependencies
+  pip:
+    name: [
+      'boto',
+      'boto3',
+      'botocore'
+    ]
+    umask: "0022"
+
+# Create temporary directory for build artifacts
+- name: Create temporary build directory
+  tempfile:
+    state: directory
+  register: tempfile
+
+
+- name: GET installer rpm from S3 bucket
+  amazon.aws.aws_s3:
+    bucket: "{{ resource_bucket_name }}"
+    object: "{{ resource_bucket_artifactory_prefix }}/jfrog-artifactory-pro-{{ artifactory_version }}.rpm"
+    dest: "/tempfile/jfrog-artifactory-pro-{{ artifactory_version }}.rpm"
```

Comment:
This won't work. It's just a hardcoded string. Have a look at [this](https://github.com/companieshouse/sonar-ami/blob/151b7bfb53314214e70d0eae80fb9979dec21b5e/ansible/roles/sonarqube/tasks/setup_sonarqube.yml#L13)